MIGRATIONS.md

```diff
@@ -1,7 +1,24 @@
 Troubleshooting Migrations
 ==========
 
-Upcoming Migrating to Version 1.4.* from 1.3.*
+Migrating to Version 1.6.* from 1.5.*
+To support Network Upgrade 5, the way keys are generated has changed.
+
+For SDK clients that regenerate the keys from a mnemonic, migration might look like:
+ * Replace usage of `UnifiedViewingKey` with `UnifiedFullViewingKey`
+ * Replace `DerivationTool.deriveUnifiedViewingKeys` with `DerivationTool.deriveUnifiedFullViewingKeys`
+
+For SDK clients that store the key separately from the mnemonic, the migration might look like:
+ * Replace usage of `UnifiedViewingKey(extfvk: String, extpub: String)` with `UnifiedFullViewingKey(encoding: String)`
+ * Replace `DerivationTool.deriveUnifiedViewingKeys` with `DerivationTool.deriveUnifiedFullViewingKeys`
+ * Delete any previously persisted values for `UnifiedViewingKey(extfvk: String, extpub: String)`.  The field `extpub` no longer exists.  The field `extfvk` needs to be re-generated from the mnemonic, and it has been renamed to `encoding` in `UnifiedFullViewingKey(encoding: String)`
```

Comment:
This migration step is incorrect; the field `extfvk` technically doesn't exist any more either. Its value happens to be contained within the `encoding` field as part of the UFVK, but that is an internal detail of UFVKs that isn't relevant to the SDK user.

Reply:
Feel free to propose alternative phrasing here or push another commit.

Reply:
Done.